src/DynamoCore/Graph/Workspaces/WorkspaceInfo.cs

```diff
@@ -148,6 +150,84 @@ public WorkspaceInfo()
             }
         }
 
+        internal static bool FromJsonDocument(String jsonDoc, string path, bool isTestMode,
```

Comment:
even though it is internal can we have some comments on what it is used for?

Reply:
Will add